orttraining/orttraining/eager/opgen/opgen/generator.py

```diff
@@ -1,6 +1,8 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
+# pylint: disable=missing-docstring, too-many-public-methods, too-many-nested-blocks
```

Comment:
I think we should aim to not disable too-many-nested-blocks since indentation matters in python and we already found one bug because of it that was very un-obvious from reading the code.